### PR TITLE
add popup modal windows

### PR DESCRIPTION
**Pull request by bengigo**
changes made in this branch: 

- store projects' information in a js object to display information dynamically
- display modal popup windows using js object

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+Array.from(projects.children).forEach((item, index) => {
+  item.firstElementChild.lastElementChild.firstElementChild.lastElementChild.firstElementChild.addEventListener(
+    'click',
+    () => {
+      modal.innerHTML = `
+        <!-- Modal content -->
+        <section class="modal-content">
+          <div class="popup-text-group">
+            <div class="text-group">
```
Comment (thecodechaser) [inline popup.js:152]: - [ ] You did a good job with the design but there is a little issue. You are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design as much as possible. Please reduce the gap between your project's sub details and make sure it matches the design on the template.

### Your design

image

### Figma design

image

File: popup.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+              <ul class="tags ${info[index].id}">
+              </ul>
+              <div class="popup-actions">
+                <button class="btn btnImg" id="btn" type="button">See Live
+                  <img src="portfolio-images/live.png" alt="icon" id="imgIcon" />
+                </button>
+                <button href='https://github.com/bengigo/micronaut-portfolio' class="btn btnImg" id="btn" type="button">See Source
+                  <img src="portfolio-images/source.png" alt="icon" id="imgIcon" />
+                </button>
+              </div>
```
Comment (thecodechaser) [inline popup.js:180]: - [ ] Please place some gap between your buttons and make sure it matches the design on the template.

### Your design

image

### Figma design

image

File: popup.js (hunk 1 above)
Comment (thecodechaser) [inline popup.js:152]: - [ ] When I open the pop-up model on the mobile screen, The close X button isn't appearing to close the popup model. Please make the close button is visible on all screens so the users can close the model.

### See the image for reference 

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @bengigo👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, 

You cover all suggestions from the previous reviewer 👍 I have some OPTIONAL suggestions also, but you should know, that your project is...

# APPROVED  👍 🥇 

It's time to merge it :shipit: after fixing OPTIONAL suggestion.

- [ ] The link to the demo live version in the README file is not updated and it is not cover your last updates with popup windows 💯 
- [ ] In the MIT.md file write down your name - just personalize it for your app 👍 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**